proposals/gc/MVP.md

```diff
@@ -266,10 +308,15 @@ TODO: Is 31 bit value range the right choice?
 
```

Comment:
Pre-existing, but: the statement "All RTT instructions are considered constant expressions" seems to be in conflict with `rtt.new`'s "multiple invocations of this instruction yield fresh RTTs".  Is that a typo or am I misinterpreting "constant expressions"?

Reply:
As per the discussion at the Lyon meeting, I resolved this to make rtt.new non-generative for now.